bin/cml.js

```diff
@@ -89,13 +107,30 @@ const handleError = (message, error) => {
           description: 'Maximum log level',
           choices: ['error', 'warn', 'info', 'debug'],
           default: 'info'
+        },
+        driver: {
+          type: 'string',
+          choices: ['github', 'gitlab', 'bitbucket'],
+          defaultDescription: 'infer from the environment',
+          description: 'Git provider where the repository is hosted'
+        },
+        repo: {
+          type: 'string',
+          defaultDescription: 'infer from the environment',
+          description: 'Repository URL or slug'
+        },
+        token: {
+          type: 'string',
+          defaultDescription: 'infer from the environment',
```

Comment:
nitpick:
```suggestion
          defaultDescription: 'infer from the environment ',
```
before: 
![image](https://user-images.githubusercontent.com/1105843/185984004-d868cfcc-b3aa-42f2-9576-aafad23d4913.png)


after:
![image](https://user-images.githubusercontent.com/1105843/185983908-1f50323b-494c-4e90-b135-810c5f1d4538.png)

Reply:
@dacbd, what's that extra whitespace for? 🤔 To avoid displaying `token[string]` without any whitespace?

Reply:
It's probably worthy of a `yargs` issue report rather than a workaround on our side (?)

Reply:
there is no space, `token[string] ...` my ocd would like to see `token [string] ...`

Reply:
obviously this is not a show stopper, very much a nitpick

Reply:
> It's probably worthy of a `yargs` issue report rather than a workaround on our side (?

probably the best option

Reply:
Let's see if your OCD is powerful enough to convert the following minimal example into an upstream issue:

```js
require('yargs').default('option', 'description', 'x'.repeat(59)).parse(['--help'])
```

If not, my own OCD will. 😄

Reply:
It did: https://github.com/yargs/yargs/issues/2228